lib/AnsiSerializer.js

```diff
@@ -14,6 +14,7 @@ Object.keys(ansiStyles).forEach(function (styleName) {
 
 function AnsiSerializer(theme) {
     this.theme = theme;
+    this.supportsColor = require('supports-color') || {};
```

Comment:
Come to think of it, wouldn't the mocking strategy still work fine if `supportsColor` were defined as `var supportsColor = require('supports-color');` at the top of `AnsiSerializer.js` rather than an instance variable? Then I'd prefer that.